lib/Imagine/Image/ManipulatorInterface.php

```diff
@@ -22,8 +22,26 @@
  */
 interface ManipulatorInterface
 {
-    const THUMBNAIL_INSET = 'inset';
-    const THUMBNAIL_OUTBOUND = 'outbound';
+    /**
+     * The original image is scaled so it is fully contained within the thumbnail dimensions (the image width/height ratio doesn't change).
+     *
+     * @var int
+     */
+    const THUMBNAIL_INSET = 0x00000001;
```

Comment:
@mlocati This BC Break is undocumented and should be added to the CHANGELOG.md

Reply:
@alexander-schranz What are you referring to with "This BC Break"?

Reply:
changing a constant value should be documented if your image configuration is not in php you can't use the constant and use the inset as string.

Reply:
You still can use the `'inset'` string: it's automatically converted to the integer value (see https://github.com/avalanche123/Imagine/pull/645/files#diff-6c8ebea98c0d67b0ee6c2083e4ab610aR114)

Reply:
Yeah, but the change should still be documented as something like: `$mode !== ManipulatorInterface::THUMBNAIL_INSET` will not longer work.

Reply:
You are right! What about editing the CHANGELOG.md file yourself and submitting a pull request?

Reply:
@mlocati I can in which version was this first released?

Reply:
This PR has been merged on 2018-08-28, and the first version after that date has been v1.0.0-alpha1
PS: reading the CHANGELOG.md, it seems that this change was already documented: see these lines:
> Add flag to thumbnail to allow upscaling images (@vlakoff)
> NOTE the $mode argument has been renamed to $settings, and it's now an integer (but old string values are accepted for backward compatibility)